examples/karaf-jersey-example/karaf-jersey-example.servicedef/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <!--
+
+        Licensed to the Apache Software Foundation (ASF) under one or more
+        contributor license agreements.  See the NOTICE file distributed with
+        this work for additional information regarding copyright ownership.
+        The ASF licenses this file to You under the Apache License, Version 2.0
+        (the "License"); you may not use this file except in compliance with
+        the License.  You may obtain a copy of the License at
+
+           http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+    -->
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>karaf-jersey-example.servicedef</artifactId>
+
+    <parent>
+        <groupId>org.apache.karaf.examples</groupId>
+        <artifactId>karaf-jersey-example</artifactId>
+        <version>4.2.2-SNAPSHOT</version>
+    </parent>
+
+    <name>Definitions of OSGi serviced used in the Jersey in Karaf demo</name>
+    <description>This is an OSGi bundle defining beans and interfaces for OSGI services.</description>
+
+    <properties>
+        <Bundle-SymbolicName>org.apache.karaf.examples.jersey.servicedef</Bundle-SymbolicName>
```

Comment:
This property doesn't seem to be use in the `maven-bundle-plugin` configuration.

Reply:
It was supposed to be, unless I've messed up something, and I think I verified what the actual symbolic name was.

The parent POM defines BundleSymbolicName to be, in properties:
```
        <Bundle-SymbolicName>${project.groupId}.${project.artifactId}</Bundle-SymbolicName>
```
and also inside the maven-bundle-plugin config (on line 301 in the top POM in my version):
```
                    <instructions>
                        <Bundle-SymbolicName>${Bundle-SymbolicName}</Bundle-SymbolicName>
```

However, when I put "-" into the artifactId I could no longer use that approach. Also the groupId contained much of the same information as the long artifactId.  So I put "overrides" in each bundle project to get a sensible symbolic name, such as eg. the example above.

I've just checked the manifest.mf of the karaf-jersey-example.servicedef-4.2.2-SNAPSHOT.jar file in my build, and it contains
```
Bundle-SymbolicName: org.apache.karaf.examples.jersey.servicedef
```
(which is the setting from the properties you quote)